content/pytorch/concepts/tensors/reshape/reshape.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: 'Pytorch Tensors Reshape'
+Description: 'Torch.reshape returns a tensor with the same data and number of elements as input, but with a specified shape.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Development'
+Tags:
+  - 'Modules'
+  - 'Methods'
+CatalogContent:
+  - 'learn-advanced-python-3'
+  - 'paths/computer-science'
+---
+
+**`Torch.reshape`** method reshapes a specified input that contains the same data and number of elements. When possible, the returned tensor will be a view of input, nevertheless, this behavior is not a guarantee. This allows the reshaping of data without copying.
+
+## Syntax
+
+`torch.reshape(input, shape)`: Returns a tensor with the same data and number of elements as input, but with a specified shape.
+
+```pseudo
+torch.reshape(input, shape)
+```
+
+## Example
+
+Using the special value `-1`:
+
+```py
+size = torch.tensor([10, 11], [12, 13])
+torch.reshape(size, (-1))
```

Comment:
```suggestion
size = torch.tensor([[10, 11], [12, 13]])
reshaped_size=torch.reshape(size, (-1))
```
Corrected the code, and added a pair of square brackets to the torch.tensor because it always uses a list of lists for clarity and consistency.
Also, I added a variable named `reshaped_size` to store the reshaped tensor and to print it. Always keep those few things in mind.

Reply:
Thank you. Unfortunately, I could not test the code.